lib/streamlit/report_session.py

```diff
@@ -629,29 +628,44 @@ def _populate_config_msg(msg: Config) -> None:
 
 
 def _populate_theme_msg(msg: CustomThemeConfig) -> None:
+    enum_encoded_options = {"base", "font"}
     theme_opts = config.get_options_for_section("theme")
 
-    if (
-        theme.check_theme_completeness(theme_opts)
-        != theme.ThemeCompleteness.FULLY_DEFINED
-    ):
+    if all([not val for val in theme_opts.values()]):
```

Comment:
Nit: `if any(theme_opts.values())`?

Reply:
Oh yeah, not sure how I didn't notice that's equivalent. Thanks!